src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const tableChild = table.children;
```

Comment:
it'll be more meaningful name
```suggestion
const rows = table.children;
```